### ISSUE TITLE
Search function results: when does highlighting happen?

### ISSUE THREAD

**Issue by hmkay** [liqd/digitalstrategie#329]
**URL:** https://digitalstrategie-dev.liqd.net/search/?q=Digitalstrategie
**user:** participant
**expected behaviour:** in the results of my search, the specific term I searched is highlightened. 
**behaviour:** Specific words are highlightened, but I do not understand why ;)

**Comment/Question:** I am comparing, btw, to the search function of the karriereportal: https://www.berlin.de/karriereportal/suche.php?site=local&q=Karriereportal

Screenshot of our search function:
<img width="760" alt="Bildschirmfoto 2022-03-23 um 19 16 18" src="https://user-images.githubusercontent.com/30862422/159768856-79f28028-863c-4872-9136-67d9f9dc79c4.png">

Screenshot of search function of Karriereportal, search for the word "Karriereportal":
<img width="760" alt="Bildschirmfoto 2022-03-23 um 19 19 48" src="https://user-images.githubusercontent.com/30862422/159769015-7888d1a7-184d-48b4-9a48-15a95c95c9fd.png">

**Comment by phillimorland** [comment]
@hmkay @maxliqd as this story is still unfinished should we add everything to the story? i'm not sure how to proceed here

**Comment by hmkay** [comment]
@phillimorland , yes, I was also unsure about the procedure here - whatever suits best for you! I can totally copy it and bring it to the story!

**Comment by goapunk** [comment]
it finds the words because they are similar in a sense. But I guess it finds too much. I guess we need to decide how this should work: for example: if you enter ``terlin``, should it find ``berlin``, if you enter ``rlin`` should it find ``berlin``, etc.. or if it should just match the exact search term and nothing else. That is definitely the simplest solution, but also won't find something if you have a typo

**Comment by phillimorland** [comment]
@hmkay we discussed in standup and we will collect issues here as the purpose of pulling that story in was to fix the styling.

**Comment by hmkay** [comment]
@goapunk I think the simples solution will do here, so let´s go for the exact search term. If there is a need to adapt it, it has to be prioritised and done later.
